PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/tree/TreeManager.java

```diff
@@ -88,6 +91,38 @@ public void initialize(URL location, ResourceBundle resources) {
   }
 
   /**
+   * Set the selection manager and initialize listeners to the selection manager.
+   *
+   * @param selectionManager the selection manager.
+   */
+  private void setSelectionManager(SelectionManager selectionManager) {
+    this.selectionManager = selectionManager;
+    selectionManager.getTopGraphNode().addListener(invalid -> {
+      colorSelectedGraphNodes();
+    });
+    selectionManager.getBottomGraphNode().addListener(invalid -> {
+      colorSelectedGraphNodes();
```

Comment:
What does `invalid` reference to?
Can't you do something like `.addListener(TreeManager::colorSelectedGraphNodes)`? Correct me if I'm wrong, not entirely sure.